spec/components/previews/support_interface/provider_user_summary_component_preview.rb

```diff
@@ -0,0 +1,11 @@
+module SupportInterface
+  class ProviderUserSummaryComponentPreview < ViewComponent::Preview
+    def provider_user_summary
+      provider_user = FactoryBot.create(:provider_user,
+                                        created_at: Time.zone.now - 2.days)
+      FactoryBot.create(:provider_permissions, provider_user: provider_user)
```

Comment:
I found out recently that previews have access to dev data, so you should be able to do ```
```ruby
 provider_user = ProviderPermissions.limit(10).sample.provider_user
```